BParkingNano/production/samples_Run3.yml

```
@@ -81,3 +81,84 @@ samples:
     dataset: /BuTOpsi2sKEE20221103FIFTYM/jodedra-SUMMER22_MINIAOD-8da3c779fd247f42d8411cf96207b146/USER
     isMC: True
     dbs: phys03
+
+  BuToKee_v1_preEE:
+    dataset: /BuToKEE_SoftQCD_TuneCP5_13p6TeV_pythia8-evtgen/Run3Summer22MiniAODv3-124X_mcRun3_2022_realistic_v12-v2/MINIAODSIM
+    isMC: True
+    dbs: global
+  BuToKee_v1_postEE:
+    dataset=/BuToKEE_SoftQCD_TuneCP5_13p6TeV_pythia8-evtgen/Run3Summer22EEMiniAODv3-124X_mcRun3_2022_realistic_postEE_v1-v2/MINIAODSIM
```

There's a small typo here, `dataset=` instead of `dataset: `, that was an issue when I sent in some test jobs.

```suggestion
    dataset: /BuToKEE_SoftQCD_TuneCP5_13p6TeV_pythia8-evtgen/Run3Summer22EEMiniAODv3-124X_mcRun3_2022_realistic_postEE_v1-v2/MINIAODSIM
```